.kokoro/upload-docs.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+# build/test docs (Node 8.16.0 is currently installed on Python image).
+export NPM_CONFIG_PREFIX=/home/node/.npm-global
+npm install
+npm run docs-test
```

Comment:
If things have hit master, I kind of want to assume this test has already run.  I'm worried with this here, it'll fail silently and we just won't know 🤷‍♂

Reply:
seems reasonable, removed tests.